core/src/main/java/com/netease/arctic/UnifiedCatalog.java

```diff
@@ -42,4 +37,9 @@ public interface UnifiedCatalog extends AmoroCatalog {
    * Refresh catalog meta
    */
   void refresh();
+
+  /**
+   * name of this catalog
+   */
+  String name();
```

Comment:
Can you keep the origin order of the methods? I cannot find the reason we should change it.